profiles/views.py

```diff
@@ -26,3 +33,13 @@ class ProfileViewSet(mixins.RetrieveModelMixin, mixins.UpdateModelMixin, viewset
     serializer_class = ProfileSerializer
     queryset = Profile.objects.all()
     lookup_field = 'user__username'
+
+
+@cache_page(60*60*24)
+def initialized_avatar(request, username, size, color, bgcolor):  # pylint:disable=unused-argument
```

Comment:
`initialized` can be a very confusing term here (it was for me at least!). Can we change this name to `name_initials_avatar_view` or something like that?